defaults/main.yml

```diff
@@ -11,6 +11,7 @@ db_name: "{{ project_name }}_{{ env_name }}"
 db_user: "{{ project_name }}_{{ env_name }}"
 db_host: 'localhost'
 db_port: 5432
+cloud_staticfiles: true
```

Comment:
Should we default to ``false``?

Reply:
doh!  Yes, you are right.  What was I thinking?